src/actions/actions.ts

```diff
@@ -2112,8 +2112,8 @@ class CommandGoToDefinition extends BaseCommand {
 
 @RegisterAction
 class CommandInsertAtFirstCharacter extends BaseCommand {
-  modes = [ModeName.Normal];
-  mustBeFirstKey = true;
+  modes = [ModeName.Normal, ModeName.Visual];
+  mustBeFirstKey = false;
```

Comment:
@johnfn I am definitely not sure...but I know it is harming this particular action

I am not sure what else I can do other than run the tests that pass and cross my fingers..